### ISSUE TITLE
Uninstallation Problem

### ISSUE THREAD

**Issue by MUGDHAB15** [Phaiax/Key-n-Stroke#104]
I am unable to uninstall it. Please provide documentation around it, as it is not visible in control panel as well

**Comment by hced** [comment]
It's not an installed app, aka portable app, so it shouldn't install anything on your system even though I'm unsure how it saves its settings. To uninstall, just move the app to the trash.

**Comment by Phaiax** [comment]
Correct, the application does not install itself, it's just the exe you downloaded. 

There is one additional small text file containing the settings in `C:\Users\username\Appdata\Local\Key-n-Stroke\settings.json`. You can reach that folder most easily by typing `%LOCALAPPDATA%` in the windows file Explorer address bar. 


https://github.com/Phaiax/Key-n-Stroke/blob/3a37f3d887850b26fdab1a47e81c0d12860460a3/KeyNStroke/SettingsStore.cs#L251
